porcupine/settings.py

```diff
@@ -841,7 +841,18 @@ def settings_to_gui(junk: object = None) -> None:
         if panedwindow["orient"] == "horizontal":
             panedwindow.paneconfig(pane, width=global_settings.get(option_name, int))
         else:
-            panedwindow.paneconfig(pane, height=global_settings.get(option_name, int))
+            """Use previous height as long as it's not less than two lines high,
+            otherwise set height to two lines high.
+            """
+            previous_height = global_settings.get(option_name, int)
+            linespace = tkinter.font.Font(font="TkFixedFont").metrics("linespace")
+            linespace += 3
```

Comment:
Maybewe could use the `get_padding()` function defined in `textutils.py` instead of a hard-coded 3?